bigquery/gcloud/aio/bigquery/bigquery.py

```diff
@@ -90,3 +90,4 @@ def _make_insert_body(rows: List[Dict[str, Any]],
         resp = await session.post(url, data=payload, headers=headers,
                                   params=None, timeout=timeout)
         resp.raise_for_status()
+        return await resp.json()
```

Comment:
Please also update this function's [type hint](https://github.com/talkiq/gcloud-aio/pull/91/files#diff-1e69c56cdf8bd8bf1742278093d488edR68): should now be `-> Dict[str, Any]`, I believe.

Reply:
I've updated the signature to a list of insertion errors. I think that's the most useful approach - there's not much of interest in the list otherwise.